backend/src/packages/users/user.repository.ts

```diff
@@ -26,6 +26,17 @@ class UserRepository implements IRepository {
     return Promise.resolve([]);
   }
 
+  public async findByPhone(value: string): Promise<UserEntity[]> {
```

Comment:
is there opportunity to get first found item in drizzle?

Reply:
no, I haven't found such possibility